UnityProject/Assets/Scripts/UI/Core/OptionsMenu/GameplayOptions.cs

```diff
@@ -17,5 +17,10 @@ public void OnPlayerExpChoiceIndexChange()
 		{
 			ProtipManager.Instance.SetExperienceLevel((ProtipManager.ExperienceLevel)playerExperienceChoices.value);
 		}
+
+		public void OnPlayerClick3D()
+        {
```

Comment:
I got dizzy trying to understand the indentation here

Reply:
indentation looks fine in  Ryder?

Reply:
make sure you are not combining tabs and spaces

Reply:
See if that worked